world.js

```diff
@@ -32,11 +32,26 @@ const localMatrix = new THREE.Matrix4();
 const localMatrix2 = new THREE.Matrix4();
 const localRaycaster = new THREE.Raycaster();
 
+let pendingAnalyticsData = {};
+setInterval(() => {
+  Object.keys(pendingAnalyticsData).map(item => {
```

Comment:
`forEach` is preferred when the result of the map is not needed.

Reply:
774e36a